### PR TITLE
Remove cross-org build step

### PR DESCRIPTION
**Pull request by sebastienros**
Matching PR: https://github.com/dotnet/arcade/pull/11577

### CODE REVIEW

**Comment by sebastienros** [comment]
Signing validation doesn't work anymore with this step since the variable group was removed

File: eng/common/templates/post-build/common-variables.yml [hunk 1]
```diff
@@ -1,8 +1,4 @@
 variables:
-  - group: AzureDevOps-Artifact-Feeds-Pats
```
Comment (sebastienros) [inline eng/common/templates/post-build/common-variables.yml:2]: An arcade update re-added, this is wrong

**Review by BrennanConroy — COMMENTED** [review]
Update publish-build-assets.yml too? Should this repo get arcade updates automatically?

**Review by BrennanConroy — APPROVED** [review]

**Comment by sebastienros** [comment]
> Should this repo get arcade updates automatically?

It does, and that's the issue. It syncs with release/7.0 which still has these variables defined, but they are removed from ADO. So the build would fail. I had removed it before but then a sync added it back and I didn't pay attention.

**Pull request merged**